libiocage/lib/helpers.py

```diff
@@ -98,7 +98,8 @@ def _prettify_output(output):
 
 # helper function to validate names
 def validate_name(name):
-    validate = re.compile(r'[a-z0-9][a-z0-9\.-_]{0,31}', re.I)
+    validate = re.compile(r'[a-z0-9][a-z0-9\.-_-]{0,31}', re.I)
```

Comment:
so, i think we want this:
```
validate = re.compile(r'[a-z0-9][a-z0-9\.\-_]{0,31}', re.I)
```

before, the regex meant: from `.` to `_`
you changed it to mean: from `.` to `_`, `-`
what we want is: `.`, `_`, `-`.